ietf/name/factories.py

```diff
@@ -0,0 +1,31 @@
+# Copyright The IETF Trust 2023, All Rights Reserved
+# -*- coding: utf-8 -*-
+
+import factory
+
+from .models import (
+    SourceFormatName,
+    StdLevelName,
+    StreamName,
+    TlpBoilerplateChoiceName,
+)
+
```

Comment:
May want to run this through Black - it'll create additional CRs between classes